crates/chain-config/Cargo.toml

```diff
@@ -14,13 +14,13 @@ description = "Fuel Chain config types"
 anyhow = "1.0"
 bech32 = "0.9.0"
 bincode = "1.3"
-fuel-core-poa = { path = "../services/consensus_module/poa", version = "0.15.1" }
 fuel-core-storage = { path = "../storage", version = "0.15.1" }
 fuel-core-types = { path = "../types", version = "0.15.1", features = [
     "serde",
     "random",
 ] }
 hex = { version = "0.4", features = ["serde"] }
+humantime-serde = "1.1.1"
```

Comment:
Is this actually used anywhere? I removed the dep and ran `ci_checks.sh` locally and everything seemed to pass for me

Reply:
Nope, it is not used, I don't know why we didn't get a warning=D

Reply:
Oh, I know=D 
<img width="688" alt="image" src="https://user-images.githubusercontent.com/18346821/213953834-8d157a48-95a8-4f07-b0d1-e736a88246d5.png">